scalableminds/webknossos
# Use displayName of organizations in gallery view

### URL of deployed dev instance (used for testing):
- https://usedisplaynameingallery.webknossos.xyz

### Steps to test:
- open gallery (you can patch line 98 in gallery_dataset_view to `showOrganizationHeader={true}`)
- ensure that the display names of the organizations is used

------
